ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml

```diff
@@ -159,23 +159,40 @@ objects:
         trap 'jobs -p | xargs -r kill || true; exit 0' TERM
 
         function fips_check() {
-          oc --insecure-skip-tls-verify --request-timeout=60s get nodes -o jsonpath --template '{range .items[*]}{.metadata.name}{"\n"}{end}' > /tmp/nodelist
-          while IFS= read -r i; do
-            oc -n default --insecure-skip-tls-verify --request-timeout=60s debug --image centos:7 node/$i -- cat /proc/sys/crypto/fips_enabled > /tmp/enabled
+          get_nodes=$(oc --request-timeout=20s get nodes -o jsonpath --template '{range .items[*]}{.metadata.name}{"\n"}{end}')
+          nodes=( $get_nodes )
+          # bash doesn't handle '.' in array elements easily
```

Comment:
While this is correct, this looks a lot different than the current style and adds a lot of verbosity.  Is this correct:

1. You added a retry loop for the debug command
2. You added verbosity to the reasons for the failure
3. You changed the request timeout

Was 3 necessary?

Reply:
no, request-timeout change was not necessary, i'll set that back to avoid confusion.

Reply:
done